SeaPublicWebsite/Views/Error/PageNotFound.cshtml

```diff
@@ -0,0 +1,34 @@
+﻿@{
+    ViewBag.Title = "Page not found - Home Energy Retrofit Advice - GOV.UK";
+}
+
+@section BeforeMain {
```

Comment:
If this is empty can you just leave it out?

Reply:
Yes i can